src/pages/todoSetup/reducers/actionTypes/todoList.ActionTypes.js

```diff
@@ -0,0 +1,7 @@
+const ACTION_TYPES = {
```

Comment:
As we have this file inside actionTypes folder, adding actionTypes again in the file name is not needed. Or add some more context then.